.github/workflows/update-wiki.yml

```
@@ -0,0 +1,92 @@
+    - 'Resources/Prototypes/Reagents/**.yml'
+    - 'Resources/Prototypes/Chemistry/**.yml'
+    - 'Resources/Prototypes/Recipes/Reactions/**.yml'
+    - 'RobustToolbox/'
+
+jobs:
+  update-wiki:
+    name: Build and Publish JSON blobs to wiki
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Master
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)